Add SITCOM-1810 Temperature sensors analysis notebook

kpenaramirez opened the pull request:
Hi Brian,
Here I allocate the temperature sensors analysis in case it is easier for your review.
Thank you in advance.
Karla

kpenaramirez committed:
SITCOM-1810 Temperature sensors analysis reviewed

bstalder reviewed (changes requested):
intro has incorrect range for thermal limits (for IQ) which is +1C/-2C with respect to the local ambient air temperature.  should also confirm everywhere (except within the mirrors where we want isothermal) that we're taking deltas between surfaces and air, not surface to surface, I found several instances where this was the case (example M2 tangent links, camrot motors).

section 1:  what is wrong with the dome telemetry?  is there a ticket for this?

section 2: top end chiller 502 is on the return side, near the camera utility trunk, not near the heat exchangers.  near the output of the heat exchangers are 504 and 505.  also to avoid confusing, TMA dynalene chiller 1 is not the correct nomenclature, should just be the TMA dynalene chiller (2) or Test dynalene chiller (1). 

section 4:  should not be using 113 for the target sensor.  it's not mounted properly yet.  please use 112 instead.

kpenaramirez committed:
SITCOM-1810_Additional_comments_addressed

kpenaramirez commented:
Hi @bstalder, I addressed your comments. Please let me know if you need me to modify something else before being able to merge.
Thank you for your time.
Karla.

kpenaramirez commented:
Hi @bstalder this is the same code you approved but instead of the thermal model for ComCam data, this is for LSSTCam data. All your comments were addressed. Please let me know if there is something missing to be able to merge.
Thank you for your time.
